reference/open223-resources.md

```diff
@@ -6,34 +6,68 @@ The [Open223 GitHub organization](https://github.com/open223) was created in con
 ## [Open223 Models](https://models.open223.info/)
 
 This site contains the canonical sources for example models.
+Each page listed in the table of contents (on the left-hand side of the page) describes an example building or system modeled in 223.
+On each page you will find the following:
+- short description of what is being modeled
+- supplementary materials and/or diagrams, where applicable
+- download links for the model (both Turtle and JSON-LD are provided)
+- links to example queries against the model; queries are hosted on the [Open223 query page](https://query.open223.info)
+- a table of entities found inside the model
 
 ## [Open223 Explore](https://explore.open223.info/)
 
-This page hosts autogenerated documentation of the 223P ontology, emphasizing the availability and correct use of concepts and properties defined by the ontology.
+This page hosts autogenerated documentation of the 223 ontology, emphasizing the availability and correct use of concepts and properties defined by the ontology.
+The Explore tool divides the ontology up across four tabs.
 
-### Navigation
+![](/_static/images/explore-tabs.png)
 
-This section will describe how to navigate the site, which allows one to traverse the standard. It uses a hierarchical structure based on classes within the standard for navigation. This section will describe how to use this page to explore standard 223P.
+#### Classes
+The classes tab organizes the classes of the standard based on their class hierarchy.
+One can click on the class name to view a description of the class and its constraints, or click on the "+" to the left of the class name to view all the subclasses of that class.
```

Comment:
```suggestion
One can click on the class name to view a description of the class and its constraints, or click on the ➕ to the left of the class name to view all the subclasses of that class.
```

Reply:
or `+`

Reply:
I'd rather not have this be a unicode character to aid with screenreaders, etc